### DISCUSSION TITLE
Gradually moving packages to Arch/Extra

### DISCUSSION THREAD

**Discussion by nwg-piotr** [nwg-piotr/nwg-shell#375] [Changelog]
I'm proud to announce migration of nwg-shell components to Arch/Extra. 

Packages: 

azote, gopsuinfo, nwg-bar, nwg-clipman, nwg-displays, nwg-dock, nwg-dock-hyprland, nwg-drawer, nwg-hello, nwg-menu, nwg-readme-browser, nwg-shell, nwg-shell-config, nwg-shell-wallpapers 

are now in Extra/Testing, and will disappear from AUR in coming days, when some required adjustments are done.

Credit goes to @Antiz96 - many thanks for making this possible.

**Comment by Antiz96** [comment]
:saluting_face: 
Thanks for those great pieces of software :smile:
